### DISCUSSION TITLE
laser engraves low porency only.

### DISCUSSION THREAD

**Discussion by Rwgiesbrecht** [arkypita/LaserGRBL#2129] [General]
hello everyone. I am new to the subject of laser and engraving, and I recently bought a laser vevor 5.5 and at first it worked very well for me. And when I did the Paso de Marco to see the engraving area, that route was also marked there. Afterwards, I tried to lower the Power to the frame and now it has no more recording power. Although I have maximum power and slow speed, it does not record me as deeply as at the beginning. Can you tell me what could be my problem?

**Comment by Tribo-2dot0** [comment]
O que você fez para diminuir a potência do enquadramento? Editou o valor no botão?

**Reply by Rwgiesbrecht** [reply]
yes that's how it is. The button was set to 3% power and I lowered it to 1%. but after giving 3 or more again, it still doesn't work for me.

**Reply by Tribo-2dot0** [reply]
A tradução está meio confusa aqui... poderia mostrar todas as suas configurações grbl e um print da caixa de edição do botão?

**Reply by Rwgiesbrecht** [reply]
![Captura de pantalla (11)](https://github.com/arkypita/LaserGRBL/assets/144503308/e423bce4-932d-4c79-89d0-0595a42189b0)
![Captura de pantalla (10)](https://github.com/arkypita/LaserGRBL/assets/144503308/50dfed30-e9af-499d-980c-312565487e56)
![Captura de pantalla (9)](https://github.com/arkypita/LaserGRBL/assets/144503308/2fd03184-d49c-4439-b7b7-501c0775eea0)
![Captura de pantalla (8)](https://github.com/arkypita/LaserGRBL/assets/144503308/617e1059-1663-41b1-aad4-abbf46d624a7)
![Captura de pantalla (7)](https://github.com/arkypita/LaserGRBL/assets/144503308/855a78fb-45e8-45f1-9543-6eec57b8634f)
![Captura de pantalla (6)](https://github.com/arkypita/LaserGRBL/assets/144503308/576c1450-d07b-4515-a077-f08e53d00052)
![Captura de pantalla (5)](https://github.com/arkypita/LaserGRBL/assets/144503308/b7491e42-29ad-4b6a-bfd3-4332968f34bf)
![Captura de pantalla (4)](https://github.com/arkypita/LaserGRBL/assets/144503308/5793012d-d600-42a2-8e0c-81c0f85a6442)
![Captura de pantalla (13)](https://github.com/arkypita/LaserGRBL/assets/144503308/bf9d62fe-9ba3-4411-9cc2-615f6e5a8021)
![Captura de pantalla (12) (1)](https://github.com/arkypita/LaserGRBL/assets/144503308/686d6206-ec5c-44b2-a026-625f32fb6df7)

good day. Here I send you the configurations.

**Comment by Tribo-2dot0** [comment]
O botão Frame está configurado para 3% de poder. Seu $30 está correto que é 1000. Se seu laser não está gravando verifique a parte de hardware, cabos e placas.

**Reply by Rwgiesbrecht** [reply]
okay. I'm going to check those parts thank you

**Reply by Rwgiesbrecht** [reply]
Well. I tried more with the machine part of the cables, changed the source to test and everything was the same. One thing I don't know if it's right. We did a test with a tester and the laser head only reaches about 6V. and on the Head it says that it is 12V. that is normal?

**Reply by Tribo-2dot0** [reply]
Os fios preto e vermelho que vão para o módulo tem que dar 12v... se possível teste com outra placa controladora.

**Reply by Rwgiesbrecht** [reply]
Well. I'm going to continue watching that. At the moment I don't have time to change the plate.
